### PR TITLE
task: adding new jobs to queue with status

### PR DESCRIPTION
**Pull request by WanProduction**

### CODE REVIEW

**Review by pirasunt — APPROVED** [review]

File: ERP/routes/web.php [hunk 1]
```diff
@@ -39,14 +39,19 @@
...
+// Route::get('/jobs', function(){
+//     return view ('jobs');
+// })
+// ->middleware('auth')
+// ->name("inventory");
+
+// Route::post('/jobs', [JobController::class, 'createJob'])
+// ->name('create.job');
+
+Route::get ('/jobs', [JobController::class, 'goToJobManagement']);
```
Comment (CeliaCaii) [inline ERP/routes/web.php:51]: Add ->middleware('auth') to your routes

**Review by xujulien99 — APPROVED** [review]
Looks good

File: ERP/routes/web.php [hunk 2]
```diff
@@ -39,14 +39,19 @@
...
+Route::get('/create-job', [JobController::class, 'goToCreateJob']);
+Route::post('/create-job', [JobController::class, 'createJob'])
+    ->name('create.job');
```
Comment (CeliaCaii) [inline ERP/routes/web.php:54]: Add **->middleware('auth')** to your routes
Maybe add -**>name()** [but this is optional, its useful for when you want to use the route() method]

**Review by mlee97 — APPROVED** [review]
Looks good!

**Pull request merged**